integration-tests/account-features-e2e.test.js

```diff
@@ -0,0 +1,45 @@
+import {Wallet, WalletDocument} from '@docknetwork/wallet-sdk-core/lib/modules/wallet';
+import {Accounts} from '@docknetwork/wallet-sdk-core/lib/modules/accounts';
+import dock from '@docknetwork/sdk';
+import { getKeyringPair } from '@docknetwork/wallet-sdk-core/lib/services/keyring';
+import { ApiRpc } from '@docknetwork/wallet-sdk-core/lib/client/api-rpc';
+
+
+
+const mockDocuments: WalletDocument[] = [];
+
+
+
+describe('Wallet integration test', () => {
+  let wallet = Wallet.getInstance();
+  let accounts = Accounts.getInstance();
+
+  beforeAll(async() => {
+      await wallet.load();
+      await Promise.all(mockDocuments.map(doc => wallet.add(doc)));
+  });
+
+  it('Import account + receive tokens + send tokens', async () => {
```

Comment:
technically, you should probably split those tests, but I can see why you might want to merge them.

It looks like this test isn't finished yet - it only appears to validate the import part so far.

Reply:
Yeah we should add some additional tests here. https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871